util/bitmap/concurrent_test.go

```diff
@@ -14,22 +14,17 @@
 package bitmap
 
 import (
+	"github.com/stretchr/testify/assert"
 	"sync"
 	"sync/atomic"
 	"testing"
-
-	. "github.com/pingcap/check"
 )
```

Comment:
These imports violate the goimports rule. If you use GoLand, you may config as
![image](https://user-images.githubusercontent.com/18818196/125961505-8815d276-96a7-49ca-9616-8a80edd231bc.png)

Reply:
Also check_dev failed on `go fmt`. You can run `go fmt ./...` from the root path of tidb repo and submit the change set.

Reply:
Updated